mapbox/styles/style.json

```diff
@@ -37,7 +37,7 @@
     }
   },
   "sprite": "https://tiles.staging.ehrenamtskarte.app/sprites/osm-liberty",
-  "glyphs": "https://orangemug.github.io/font-glyphs/glyphs/{fontstack}/{range}.pbf",
+  "glyphs": "https://tiles.staging.ehrenamtskarte.app/font-glyphs/glyphs/{fontstack}/{range}.pbf",
```

Comment:
```suggestion
  "glyphs": "https://tiles.staging.ehrenamtskarte.app/glyphs/{fontstack}/{range}.pbf",
```

Reply:
@maxammann Are you sure? If we just serve the styles directory, then there is this subfolder font-glyphs on top of glyphs (as glyphs is not the only child of font-glyphs).

Reply:
Done